api/v1alpha1/postgresqlserviceuser_types.go

```diff
@@ -17,17 +17,15 @@ limitations under the License.
 package v1alpha1
 
 import (
+	apiv1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
-
 // PostgreSQLServiceUserSpec defines the desired state of PostgreSQLServiceUser
 // +k8s:openapi-gen=true
 type PostgreSQLServiceUserSpec struct {
 	// Name of the service user
-	Name string `json:"name"`
+	Username string `json:"username"`
```

Comment:
Consider if `username` also should be of type `ResourceVar` so all of it can come from a secret.